packages/backend/common/models/user.js

```diff
@@ -20,4 +22,5 @@ const schema = Joi.object({
   clubs: Joi.array().items(Joi.string())
 })
 
-module.exports = schema
+module.exports = schema;
+module.exports.id = id;
```

Comment:
Instead of exporting a separate variable like on line 26. You can do
```
schema.id = id
```

Then on `club.js`, you can do UserSchema.id. Now we can do that syntax because we just added the id property into UserSchema.

If not doing that and keep your remaining code the same, on line 13 of `club.js`, it has to be
```
officers: Joi.array().items(id).min(1),
``` 
due to id being export on top (line 2)

Reply:
Ok, Ill take a look